chsdi/templates/htmlpopup/kgs_anhoerung.mako

```diff
@@ -0,0 +1,151 @@
+        </tr>
+        <tr>
+            <th class="cell-left">${_('tt_kbs_nbr')}</th>
+            <td>${c['featureId'] or '-'}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('grundadresse')}</th>
+            <td>${c['attributes']['adresse'] or ''} ${c['attributes']['hausnr'] or ''}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('tt_kbs_gemeinde')} (${_('tt_kbs_gemeinde_ehemalige')})</th>
+            <td>${c['attributes']['gemeinde'] or ''} ${'('+c['attributes']['gemeinde_ehemalig']+')' if c['attributes']['gemeinde_ehemalig'] else ''}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('Coordinates')}</th>
+            <td>${int(round(c['attributes']['x'],0)) or ''} / ${int(round(c['attributes']['y'],0)) or ''}</td>
+        </tr>
+    % if c['attributes']['pdf_list'] is not None:
+        <tr>
+            <th class="cell-left">${_('Feature tooltip')}:</th>
+            <td>
+	        % for pdf in c['attributes']['pdf_list'].split(','):
+                <a href="https://${dataGeoAdminHost}/${dataPath}/PDF/${pdf}" target="_blank">${pdf}</a><br />
+	        % endfor
+            </td>
+	    </tr>
+    %endif
+    % if c['attributes']['link_uri'] is not None:
+        <tr>
+          <th class="cell-left">${_('legalregulationlink')}</th>
+            <td><a href="${c['attributes']['link_uri']}">${c['attributes']['link_title']}</a></td>
+        </tr>
+    % endif
+    % if c['attributes']['link_2_uri'] is not None:
+        <tr>
+          <th class="cell-left">${_('legalregulationlink')}</th>
+          <td><a href="${c['attributes']['link_2_uri']}">${c['attributes']['link_2_title']}</a></td>
+        </tr>
+    % endif
+    % if c['attributes']['link_3_uri'] is not None:
+        <tr>
+          <th class="cell-left">${_('legalregulationlink')}</th>
+          <td><a href="${c['attributes']['link_3_uri']}">${c['attributes']['link_3_title']}</a></td>
+        </tr>
+    % endif
+    </table>
+     <div id="blueimp-gallery" class="blueimp-gallery blueimp-gallery-controls">
+        <div class="slides"></div>
+        <div class="title" id="blueimp-gallery-title"></div>
+        <a class="prev">&lsaquo;</a>
+        <a class="next">&rsaquo;</a>
+        <a class="close">x</a>
+        <a class="play-pause"></a>
+        <ol class="indicator"></ol>
+     </div>
+        <div class="kgs-thumbnails">
+            <div class="thumbnail-container">
+            %for pic in pic_list:
+                <div class="thumbnail">
+                    <a href="https://${dataGeoAdminHost}/${dataPath}/image/kgs_${pic[0]}_${pic[1]}.jpg">
+                        <img class="image" src="https://${dataGeoAdminHost}/${dataPath}/image/kgs_${pic[0]}_${pic[1]}.jpg" />
+                    </a>
+                    <div>${pic[3] or ''} - ${pic[2] or ''}</div>
+                </div>
+            %endfor
+            </div>
+        </div>
+    <table class="kernkraftwerke-extended">
```

Comment:
samme comment as in line 54

Reply:
Not changing this in this pull request as it touches around 30 makos.